scripts/onboarding/aks/onboarding-using-msi-auth/existingClusterParam.json

```diff
@@ -17,6 +17,9 @@
     "enableContainerLogV2": {
       "value": true
     },
+    "enableRetinaNetworkFlags": {
```

Comment:
shouldnt we expose "enableRetinaNetworkFlowLogs" or "enableNetworkFlowLogs" though underlying field is enableRetinaNetworkFlags?